File: 13-bazel-resources/java/foo/foo.java
package foo;

import java.io.IOException;
import java.io.InputStream;
import java.nio.charset.StandardCharsets;

public class foo {
    public static void main(String args[]) throws IOException {
        InputStream is = foo.class.getResourceAsStream("/foo/resources/a.txt");
        String text = new String(is.readAllBytes(), StandardCharsets.UTF_8);
        System.out.println(text);
    }
}

File: 13-bazel-resources/scala/foo/foo.scala
package foo

import java.io._
import java.nio.charset.StandardCharsets

object Foo {
  def main(args: Array[String]) = {
    val is: InputStream = getClass.getResourceAsStream("/a.txt")
    val content = scala.io.Source.fromInputStream(is).mkString
    println(content)
  }
}
